calendar/month.go

```diff
@@ -254,7 +254,9 @@ func (m *monthView) getCalendarRows() []string {
 		for x := 0; x < len(days); x++ {
 			day := days[x]
 
-			dayStr := m.getCalendarDay(day, x, y)
+			events := GetEventsByDate(day)
+
+			dayStr := m.getCalendarDay(day, x, y, events)
```

Comment:
I think we should split this out. Loading all the events for the month before doing any rendering would allow us to do lookups on the event id. That way we could check the events for the previous day and see if the event continues.